src/auth/interceptors/createCsrfTokenProviderInterceptor.js

```diff
@@ -0,0 +1,39 @@
+import getCsrfToken from '../getCsrfToken';
+
+const createCsrfTokenProviderInterceptor = (options) => {
```

Comment:
Renamed to `createCsrfTokenProviderInterceptor ` from `csrfTokenProviderInterceptor` to better indicate what is happening here.